### PR TITLE
✨Add streaming function / Fix spawnVehicle

### PR DESCRIPTION
**Pull request by LQuatre**

### CODE REVIEW

**Review by MoskalykA — CHANGES_REQUESTED** [review]
There are a lot of indentation and syntax errors, but most of all I don't see the point of this addition, please correct it.

**Comment by LQuatre** [comment]
Ok i will change that

**Comment by LQuatre** [comment]
Done

File: src/api/streaming/client/requestModel.lua [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---requestModel
+---@param modelHash string
+---@param callback function
+---@public
+function API_Streaming:requestModel(modelHash, callback)
+    if (not (IsModelInCdimage(modelHash))) then
+        return _NCS:die("Model not found: " .. modelHash)
+    end
+    modelHash = (type(modelHash) == 'number' and modelHash or GetHashKey(modelHash))
```
Comment (MoskalykA) [inline src/api/streaming/client/requestModel.lua:9]: use "

File: src/api/vehicles/client/spawnVehicle.lua [hunk 2]
```diff
@@ -12,21 +12,16 @@ function API_Vehicles:spawn(modelName, coords, heading, callback, clientSide)
...
+    API_Streaming:requestModel(modelName, function()
+        local vehicle = CreateVehicle(modelName, coords, heading, not (clientSide), 0)
+        SetVehicleDirtLevel(vehicle, 0.0)
+        SetEntityCoordsNoOffset(vehicle, coords.x, coords.y, coords.z + 0.5, 0.0, 0.0, 0.0)
+        SetVehicleOnGroundProperly(vehicle)
+        SetEntityHeading(vehicle, heading)
+        SetEntityAsMissionEntity(vehicle, 1, 1)
+        if callback ~= nil then
+            callback(vehicle)
+        end
```
Comment (MoskalykA) [inline src/api/vehicles/client/spawnVehicle.lua:24]: no callback

**Review by MoskalykA — CHANGES_REQUESTED** [review]

**Comment by LQuatre** [comment]
done

**Review by MoskalykA — CHANGES_REQUESTED** [review]

**Comment by LQuatre** [comment]
i need help for somethings wait next update

**Pull request merged**